packages/components/src/progress-bar/styles.ts

```diff
@@ -43,7 +43,7 @@ export const Indicator = styled.div< {
 	top: 0;
 	height: 100%;
 	border-radius: ${ CONFIG.radiusBlockUi };
-	background-color: ${ COLORS.ui.theme };
```

Comment:
Curious, was this not working? The value is supposed to be pretty much the same:

https://github.com/WordPress/gutenberg/blob/904fd2400245914b00923d05f21de3335ab69a32/packages/components/src/utils/colors-values.js#L34

Reply:
Hmm, today I can no longer reproduce the scenario. But I'd still prefer to keep it consistent with the other color styles for this component. I realize it's very unlikely the underlying `COLORS` constant to change, but still, I appreciate that when looking at the styles of the progress bar component it's immediately clear what colors we're using, without having to peek at the `COLORS` definition. Will go ahead and merge, but let me know if you have any second thoughts about it.

Reply:
Would it be better if we renamed the `COLORS.ui.theme` variable to something that evokes the underlying CSS variable, like `COLORS.theme.accent`? For example, in Sass we have it set up [like this](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/utils/theme-variables.scss).

In any case I don't think it makes sense to do `var( --wp-components-color-accent, ${ COLORS.ui.theme } )` because that renders an entirely unnecessary level of nesting:

```
var( --wp-components-color-accent, var( --wp-components-color-accent, var( --wp-admin-theme-color, #3858e9 ) ) )
```

Reply:
> in the components package, we call it accent color, but historically we've also called that color just "admin theme color".

Right. It was intentional on our part to decouple it from the wp-admin theme color. And the components `Theme` system involves other colors like background color and grayscales as well, so we can't just call it "the theme color".

> I'd be happy with the rename as you suggested it, but would it make sense for all the pre-existing `COLORS.ui.theme` use cases?

Yes 🎉  We haven't done the grayscale stuff yet, but the accent colors used in all our components are `Theme`-ready (#45289). Meaning, every use of `COLORS.ui.theme` and `COLORS.ui.themeDark10` in our codebase is already hand-verified to look as expected in a `Theme` context. So it would be perfectly ok to rename them all to `COLORS.theme.accent` and `COLORS.theme.accentDarker10` to better reflect the underlying CSS variable names.

FYI the [colors-values.js](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/utils/colors-values.js) file is not completely cleaned up yet. It's likely that we'll eventually remove most of the things in the `COLORS.ui` object, and move the rest of them to `COLORS.theme`.

Reply:
Sounds good, thanks for confirming @mirka! I'm planning to follow up on this next week!